frontend/src/metabase/admin/people/components/GroupMembersTable/GroupMembersTable.tsx

```diff
@@ -205,3 +219,24 @@ function getName(user: IUser): string {
 
   return name;
 }
+
+const ApiKeyRow = ({ apiKey }: { apiKey: ApiKey }) => {
+  return (
+    <tr>
+      <td>
+        <Text weight="bold">{apiKey.name}</Text>
+      </td>
+      <td>
+        <Text weight="bold" color="text.1">{t`API Key`}</Text>
+      </td>
+      <td>{/* api keys don't have emails */}</td>
```

Comment:
```suggestion
      <td>{/* api keys don't have user-facing emails */}</td>
```

I believe API keys do get associated with internally used email addresses with the format "api-key-user-%s@api-key.invalid", due to [this backend code](https://github.com/metabase/metabase/blob/4d5b621879173c187d28bb58b234197d31dfbe7e/src/metabase/api/api_key.clj#L55).